integration_test/busybenchmark.js

```diff
@@ -0,0 +1,18 @@
+let durationSeconds = 600;
+const startTime = Date.now();
+
+function benchmark() {
+  var buffer = new Buffer(10000);
```

Comment:
Nit: why 10000 is sometimes 10000 and sometimes 1e4?

Reply:
Done. Switched to 1e4, since it seems more readable to me.